core/src/com/unciv/ui/worldscreen/unit/UnitTable.kt

```diff
@@ -156,11 +156,7 @@ class UnitTable(val worldScreen: WorldScreen) : Table(){
             selectedUnitHasChanged = true
         }
         else {
-            separator.isVisible=false
-            unitNameLabel.setText("")
-            unitDescriptionTable.clear()
-            unitIconHolder.clear()
-            promotionsTable.clear()
```

Comment:
Is this change related to the Effel Tower effect?

Reply:
when unit is destroyed the unit table should be not visible.